### PR TITLE
Change service example

### LINKED ISSUE #241: Economy Services example needs changing.
There is significant chatter in #spongedev that an official economy service of some kind will be implemented into SpongeAPI.

I'd suggest changing the example service provided to another example that is less likely to be included in the API, so it avoids confusion.

### PR DESCRIPTION
**Pull request by Flibio**
Changes the service example from an economy service to a warp service. Closes #241.

---

[Preview the file change](http://flibio.net/preview/service-example/plugin/services/)

### CODE REVIEW

File: source/plugin/services.rst [hunk 1]
```diff
@@ -28,62 +28,67 @@ Designing the API this way makes Sponge extremely modular.
...
+        HashMap<String, Location<World>> warpMap = new HashMap<String, Location<World>>();
 
-    	@Override
-    	public int getBalance(Player player)
-    	{
-    		if(!balanceMap.containsKey(player)) return 0; //no stored balance for player
+        @Override
+        public Optional<Location<World>> getWarp(String name)
+        {
+            if(!warpMap.containsKey(name)) return Optional.empty();
```
Comment (kashike) [inline source/plugin/services.rst]: This should be formatted correctly to match the rest of the examples.

File: source/plugin/services.rst [hunk 2]
```diff
@@ -28,62 +28,67 @@ Designing the API this way makes Sponge extremely modular.
...
 
-    		else return balanceMap.get(player);
-    	}
+            else return Optional.of(warpMap.get(name));
```
Comment (kashike) [inline source/plugin/services.rst]: See above.

File: source/plugin/services.rst [hunk 3]
```diff
@@ -28,62 +28,65 @@ Designing the API this way makes Sponge extremely modular.
...
+    import org.spongepowered.api.world.World;
+    import java.util.Optional;	
 
-    public interface EconomyService
-    {
-    	public int getBalance(Player player);
-    	public void setBalance(Player player, int amount);
+    public interface WarpService {
+        public void setWarp(String name, Location<World> location);
+        public Optional<Location<World>> getWarp(String name);
```
Comment (12AwsomeMan34) [inline source/plugin/services.rst]: Isn't declaring these public pretty much redundant?

http://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4

File: source/plugin/services.rst [hunk 4]
```diff
@@ -28,62 +28,65 @@ Designing the API this way makes Sponge extremely modular.
...
     }
 
 Now you can write the class that implements your interface:
 
 .. code-block:: java
 
+    import org.spongepowered.api.world.Location;
+    import org.spongepowered.api.world.World;
```
Comment (12AwsomeMan34) [inline source/plugin/services.rst]: Location and world are already implemented in the above code block

File: source/plugin/services.rst (hunk 3 above)
Comment (kashike) [inline source/plugin/services.rst]: Yes.

**Pull request closed without merging**